src/components/MyProfile/MyProfileContainer.tsx

```diff
@@ -32,6 +38,14 @@ const MyProfileContainer = () => {
     {
       onSuccess: ({ data }) => {
         formik.setValues({ ...data.data.user, ...data.data.introduction });
+        setPosition({
+          lat: data.data.introduction.latitude,
+          lng: data.data.introduction.longitude,
+        });
+        setCenterPosition({
+          lat: data.data.introduction.latitude,
+          lng: data.data.introduction.longitude,
+        });
```

Comment:
온라인으로 설명을 들어보니

- setPosition은 사용자가 클릭하는 마커의 위치를 의미하고
- setCenterPosition은 지도를 처음 렌더링할 때 보여줄 중심 위치를 의미한다

라는 걸 알게 되었습니다.

```suggestion
        setUserClickPosition({
          lat: data.data.introduction.latitude,
          lng: data.data.introduction.longitude,
        });
        setMapCenterPosition({
          lat: data.data.introduction.latitude,
          lng: data.data.introduction.longitude,
        });
```

이렇게 설명이 더 드러나는 이름이 생각이 나는데요, 이것보다도 더 좋은 이름도 있지 않을까 싶습니다!